### ISSUE TITLE
 Ability to define a behaviour for highways 

### ISSUE THREAD

**Issue by JenMysoc** [mysociety/fixmystreet#737]
Email from Jon Kristensen of FixaMinGata:

"First of all, we want to be able to define a behaviour for
highways so that problems reported for them goes to the national
department for roads instead of the municipality in question. Also, we
want to be able to say that "any kind of roads within this geometrical
area should not go to the municipality, but to this e-mail address, or
non-municipality body, instead" (this solves a problem where road
associations are responsible for sets of roads within municipalities).
In short, we don't want municipalities to receive problem reports for
roads that they are not maintaining, be it highways or roads for which
they have delegated the responsibility to some other entity."

Similar to the UK ticket #736
